src/Moq/Extensions.cs

```diff
@@ -250,24 +250,12 @@ public static bool CanOverride(this MethodBase method)
 
 		public static bool CanOverrideGet(this PropertyInfo property)
 		{
-			if (property.CanRead)
-			{
-				var getter = property.GetGetMethod(true);
-				return getter != null && getter.CanOverride();
-			}
-
-			return false;
+			return property.CanRead(out var getter) && getter.CanOverride();
```

Comment:
This should be safe even when a base class derives a `virtual` readable property and the derived class decides to `sealed` the getter&mdash;in such a case, `property.CanRead()` will retrieve the more derived `sealed` accessor.